forkpi/spoonpi/spoonpi/forkpi_db.py

```diff
@@ -33,6 +35,20 @@ def authenticate(self, *args, **kwargs):
              This is an empty list if is_authorized is False
         """
         keywords = kwargs.keys()
+        GLOBAL_PIN = "global_pin"
+        PIN = "pin"
+        RFID_UUID = "rfid_uid"
+        FINGERPRINT_MATCHES = "fingerprint_matches"
+
+        if GLOBAL_PIN in keywords:
+            global_pin = kwargs[GLOBAL_PIN]
+            user = User.objects.filter(username=kwargs.get("username")).first()
```

Comment:
Why do we need `first()`? We should use `get()` so that if username doesn't exist, we get an error.